datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BaseStorageTest.java

```
@@ -89,4 +102,116 @@ private void validatePrivateStructAndEncryption(AbsoluteResourceLocation<Private
                 .filter(it -> pattern.test(it.location().toString()))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @SneakyThrows
+    @Ignore
+    public void WriteToPrivateListPrivateInDifferentThreads() {
+        CountDownLatch fileSaveCountDown = new CountDownLatch(NUMBER_OF_TEST_USERS * NUMBER_OF_TEST_FILES);
+        String path = "folder2";
+
+        log.trace("*** Starting write threads ***");
+        for (int i = 0; i < NUMBER_OF_TEST_USERS; i++) {
+            UserIDAuth john = registerUser("john_" + i, location);
+            log.debug("Registered user: {}", john.getUserID().getValue());
+
+            AtomicInteger counter = new AtomicInteger();
+
+            for (int j = 0; j < NUMBER_OF_TEST_FILES; j++) {
+                new Thread(() -> {
+                    try {
+                        log.trace("Start thread: {} for user: {}",
+                                Thread.currentThread().getName(), john.getUserID().getValue());
+
+                        String filePath = path + "/" + counter.incrementAndGet() + ".txt";
+
+                        log.debug("Saving file: {}", filePath);
+                        writeTextToFileForUser(john, filePath, MESSAGE_ONE, fileSaveCountDown);
+                    } catch (IOException e) {
+                        e.printStackTrace();
```

Instead of swallowing this we need to abort test